### PR TITLE
Refactor the documentation and update readme files

### PR DESCRIPTION
**Pull request by dasuni-30**
> Refactor the documentation and update readme files

### CODE REVIEW

File: petcare-enterprise-apps/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Pet Care Enterprise Applications
+
+## Pet Management Application (B2C)
```
Comment (malithie) [inline petcare-enterprise-apps/README.md]: PetDesk Application for Pet Owners

File: petcare-enterprise-apps/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+### 1. PetDesk web application 
+
+You can run the PetDesk B2C web application using cloud deployment or locally using the following guides.
+
+&ensp;&ensp;&ensp;&ensp;1. Run in cloud environment using the [guide](/petcare-enterprise-apps/docs/PETDESK-B2C-CLOUD.md)
+<br>
+&ensp;&ensp;&ensp;&ensp;2. Run Locally using the [guide](/petcare-enterprise-apps/docs/PETDESK-B2C-LOCAL.md)
+
+### 2. Pet Care mobile application
```
Comment (malithie) [inline petcare-enterprise-apps/README.md]: PetDesk mobile application

File: petcare-enterprise-apps/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+You can try the Pet Care B2C mobile application using the [guide](/petcare-enterprise-apps/docs/PETCARE-MOBILE.md)
+
+## Pet Care Veterinary Practice Management Application (B2B)
```
Comment (malithie) [inline petcare-enterprise-apps/README.md]: PetCare PetVet Veterinary Practice Management Application

File: petcare-enterprise-apps/docs/PETCARE-MOBILE.md [hunk 4]
```diff
@@ -1,23 +1,23 @@
 # Pet care sample with Asgardeo Android SDK
```
Comment (malithie) [inline petcare-enterprise-apps/docs/PETCARE-MOBILE.md]: PetCare PetDesk sample

File: petcare-enterprise-apps/docs/PETCARE-MOBILE.md [hunk 5]
```diff
@@ -1,23 +1,23 @@
...
+Required versions
  ```
  Java version >= 17
  ```
  ```
  Android Studio version >= Giraffe
  ```
 
-#### 1. Setup steps Identity Server
+# 1. Setup steps Identity Server
```
Comment (malithie) [inline petcare-enterprise-apps/docs/PETCARE-MOBILE.md]: Let's write this for Asgardeo

File: petcare-enterprise-apps/docs/PETDESK-B2C-CLOUD.md [hunk 6]
```diff
@@ -1,16 +1,4 @@
...
-There are two ways that you can run this sample application
-
-- Method 1: [Using Cloud Deployment](#️-method-1-using-cloud-deployment)
-- Method 2: [Run Locally](#️-method-2-run-locally)
-
-&nbsp;<br>
-&nbsp;<br>
-
-# ✍️ Method 1: Using Cloud Deployment
+# Run PetDesk B2C Application in cloud environment
```
Comment (malithie) [inline petcare-enterprise-apps/docs/PETDESK-B2C-CLOUD.md]: Run PetCare PetDesk B2C xxx

File: petcare-enterprise-apps/docs/PETDESK-B2C-LOCAL.md [hunk 7]
```diff
@@ -0,0 +1,179 @@
+# Run PetDesk B2C Application locally
```
Comment (malithie) [inline petcare-enterprise-apps/docs/PETDESK-B2C-LOCAL.md]: PetCare PetDesk

File: petcare-enterprise-apps/docs/PETDESK-B2C-LOCAL.md [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+
+## Prerequisites:
+1. Install Ballerina 2201.5.0 https://dist.ballerina.io/downloads/2201.5.0/ballerina-2201.5.0-swan-lake-macos-arm-x64.pkg
+2. Install Node 16 LTS (Tested in v16.13.0).
+3. Configure choreo webhook for [Asgardeo user registration event](https://wso2.com/asgardeo/docs/guides/asgardeo-events/#implement-business-use-cases-for-asgardeo-events). Use [asgardeo_registration_webhook](/petcare-enterprise-apps/b2c/webhooks/asgardeo_registration_webhook/).
+
+When deploying the webhook through choreo, provide the salesforce related configuration by getting them using this [guide](#setup-salesforce-account-guide).
+
+
+## Create an Application in WSO2 Identity Server
```
Comment (malithie) [inline petcare-enterprise-apps/docs/PETDESK-B2C-LOCAL.md]: Let's say asgardeo as tutorial is formed around asgardeo here

File: petcare-enterprise-apps/README.md (hunk 1 above)
Comment (dasuni-30) [reply petcare-enterprise-apps/README.md]: Addressed with 2dacf12da2870eaf5efbd6c93f79edf6f24d71ce.

File: petcare-enterprise-apps/README.md (hunk 3 above)
Comment (dasuni-30) [reply petcare-enterprise-apps/README.md]: Addressed with 2dacf12da2870eaf5efbd6c93f79edf6f24d71ce.

File: petcare-enterprise-apps/docs/PETCARE-MOBILE.md (hunk 5 above)
Comment (malithie) [reply petcare-enterprise-apps/docs/PETCARE-MOBILE.md]: Shouldn't we change the title here ?

File: petcare-enterprise-apps/docs/PETDESK-B2C-CLOUD.md [hunk 9]
```diff
@@ -1,16 +1,4 @@
...
-There are two ways that you can run this sample application
-
-- Method 1: [Using Cloud Deployment](#️-method-1-using-cloud-deployment)
-- Method 2: [Run Locally](#️-method-2-run-locally)
-
-&nbsp;<br>
-&nbsp;<br>
-
-# ✍️ Method 1: Using Cloud Deployment
+# Run Pet Care PetDesk B2C B2C Application in cloud environment
```
Comment (malithie) [inline petcare-enterprise-apps/docs/PETDESK-B2C-CLOUD.md]: Have two "B2C"

File: petcare-enterprise-apps/docs/PETCARE-MOBILE.md (hunk 5 above)
Comment (dasuni-30) [reply petcare-enterprise-apps/docs/PETCARE-MOBILE.md]: Added with 289d247f19c4b680f81c318068d5a6e2430cfa29

File: petcare-enterprise-apps/README.md (hunk 2 above)
Comment (dasuni-30) [reply petcare-enterprise-apps/README.md]: Added with 2dacf12da2870eaf5efbd6c93f79edf6f24d71ce.

File: petcare-enterprise-apps/docs/PETDESK-B2C-CLOUD.md (hunk 6 above)
Comment (dasuni-30) [reply petcare-enterprise-apps/docs/PETDESK-B2C-CLOUD.md]: Added with 2dacf12da2870eaf5efbd6c93f79edf6f24d71ce.

File: petcare-enterprise-apps/docs/PETDESK-B2C-LOCAL.md (hunk 7 above)
Comment (dasuni-30) [reply petcare-enterprise-apps/docs/PETDESK-B2C-LOCAL.md]: Added with 2dacf12da2870eaf5efbd6c93f79edf6f24d71ce.

File: petcare-enterprise-apps/docs/PETDESK-B2C-CLOUD.md (hunk 9 above)
Comment (dasuni-30) [reply petcare-enterprise-apps/docs/PETDESK-B2C-CLOUD.md]: Added with 68bba15641863259192a926b1b31f51c3dd30991.

File: petcare-enterprise-apps/docs/PETCARE-MOBILE.md (hunk 4 above)
Comment (dasuni-30) [reply petcare-enterprise-apps/docs/PETCARE-MOBILE.md]: Added with 2dacf12da2870eaf5efbd6c93f79edf6f24d71ce.

File: petcare-enterprise-apps/docs/PETDESK-B2C-LOCAL.md (hunk 8 above)
Comment (dasuni-30) [reply petcare-enterprise-apps/docs/PETDESK-B2C-LOCAL.md]: Added with 060ec807c496a52bde3a462dea3f0ec7c480360f

**Review by malithie — APPROVED** [review]

**Pull request merged**